src/plugins/intel_cpu/README.md

```diff
@@ -0,0 +1,24 @@
+# CPU Plugin
+
+## Components
+
+CPU Plugin contains the following components:
+
+* [src](./src/)
```

Comment:
Please add the description for each component

Reply:
Done [834da8d](https://github.com/openvinotoolkit/openvino/pull/15372/commits/834da8d18b9e53191821e842b80ef3b8f1586290)